### ISSUE TITLE
Spigot mapping support

### ISSUE THREAD

**Issue by Titaniumtown** [jaredlll08/MCBot#67]
Would it be possible to add support for spigot's mappings as well? This discord bot seems very useful, but the thing is I would need to be able to use spigot's mappings as well. Is this something that could possibly be added?

**Comment by tterrag1098** [comment]
Yes, but probably not by me. The mappings API is extendable (see #63) so it should be trivial to add more mappings sources.
